tests/generic_relations/tests.py

```diff
@@ -405,6 +405,13 @@ def test_generic_relation_to_inherited_child(self):
         # GenericRelations to models that use multi-table inheritance work.
         granite = ValuableRock.objects.create(name='granite', hardness=5)
         ValuableTaggedItem.objects.create(content_object=granite, tag="countertop", value=1)
+        self.assertEqual(ValuableRock.objects.filter(tags__value=1).count(), 1)
+        # We are generating a big inefficient query for tags__tag - we join first
```

Comment:
I'm not sure the purpose of this comment -- is it describing something that could be improved or just the fact that the implementation isn't very elegant (and probably can't be)?